cache-caffeine/src/test/groovy/io/micronaut/cache/client/CachingReactorCrudSpec.groovy

```diff
@@ -55,25 +57,27 @@ class CachingReactorCrudSpec extends Specification {
         given:
         BookClient client = context.getBean(BookClient)
         BookController bookController = context.getBean(BookController)
-
+        
         when:
-        Book book = client.get(99).block()
-        List<Book> books = client.list().block()
+        Optional<Book> bookOptional = Mono.from(client.get(99))
+                .onErrorResume(t -> Mono.empty())
```

Comment:
since this is throwing a not found exception I have wrapped in an `onErrorResume` and `blockOptional`